wagtail/tests/settings.py

```diff
@@ -136,5 +137,7 @@
 except ImportError:
     pass
 
+# Sendfile dev backend, do NOT use in production https://github.com/johnsensible/django-sendfile#django-sendfile
+SENDFILE_BACKEND = 'sendfile.backends.development'
```

Comment:
Looks like this is now a required setting: https://github.com/johnsensible/django-sendfile/blob/master/sendfile/__init__.py#L27-L29